Packs/MISP/ReleaseNotes/1_0_5.md

```diff
@@ -0,0 +1,4 @@
+
+#### Integrations
+##### MISP v2
+- **Breaking Change**: Added a parameter **Maximum attributes in event** to limit attributes per event in context output. Parameter is set to 1000 by default. This limit was added in order to prevent issues where large MISP events created high memory and disk usage.
```

Comment:
```suggestion
- **Breaking Change**: Added the parameter **Maximum attributes in event** to limit attributes per event in the context output. The parameter is set to 1000 by default. This limit was added in order to prevent issues where large MISP events would create high memory and disk usage.
```